reinforcement_learning/Lesson-6-PPO-draft.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>Expand on the intuition and goal of PPO. See HF for a good example: <a href="https://huggingface.co/learn/deep-rl-course/unit8/intuition-behind-ppo" target="_blank">https://huggingface.co/learn/deep-rl-course/unit8/intuition-behind-ppo</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Still room for improvement here. Particularly the explanations for the why and how components of the goal of PPO. The flow can improve as well.  The HF explanation of PPO's goal is nearly perfect. We can use theirs and adjust it to our audience a bit. We would obviously credit them by providing a reference link with an associated comment. This would be similar to what HF did for their explanation, which is an adaption of the explanation provided by Jonathan Hui, i.e., HF added this: "Modified version from RL — Proximal Policy Optimization (PPO) [Explained by Jonathan Hui](https://jonathan-hui.medium.com/rl-proximal-policy-optimization-ppo-explained-77f014ec3f12)".